src/components/menu/API.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import {Trans, withTranslation, WithTranslation} from "react-i18next";
+import {IAlert} from "../../types";
+
+
+interface IState {
+    alerts: IAlert[];
+}
+
+class API extends React.Component<WithTranslation, IState> {
+
+    state = {alerts: []};
+
+    public render() {
+    
+        return (
+            <div>
+                <div className="content box" id="API">
+                    <h3><Trans>API</Trans></h3>
+                        <h4>Preset</h4>
+                        <h5>Preset</h5>
+                            <p>Request the selected preset.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/preset</dd>   
+                                <dt>Request Parameters</dt>
+                                <dd>id</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/preset/simple</dd>
+                                <dd>
+                                <ul>
+                                    <li>TECHNICAL = 0</li>
+                                    <li>AOK = 1</li>
+                                    <li>CONQUERORS = 2</li>
+                                    <li>FORGOTTEN = 3</li>
+                                    <li>AFRICAN_KINGDOMS = 4</li>
+                                    <li>RISE_OF_RAJAS = 5</li>
+                                    <li>DEFINITIVE_EDITION = 6</li>
+                                    <li>LORDS_OF_THE_WEST = 7</li>
+                                    </ul>
+                                </dd>
+                            </dl>
+
+                            <h5>Add Preset</h5>
+                            <p>Request the selected preset.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/preset</dd>   
+                                <dt>Request Parameters</dt>
+                                <dd>No Clue</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/preset/new</dd>
+                            </dl>
+
+                            <h5>List Preset</h5>
+                            <p>Lists all presets with name and id.</p>
```

Comment:
It does not list all presets, just the ones from a public list.

Reply:
changed to Lists public presets...